salt/states/x509.py

```diff
@@ -203,7 +203,8 @@ def _revoked_to_list(revs):
 def private_key_managed(name,
                         bits=2048,
                         new=False,
-                        backup=False):
+                        backup=False,
+                        verbose=False,):
```

Comment:
Same comment here about the default value. We should default this to True so we don't surprise people who may be relying on the current functionality.